src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,23 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+type Togo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
```

Comment:
The type `Togo` should be renamed to `Todo` to maintain consistency with the naming conventions used in a todo application.